package.json

```diff
@@ -18,10 +18,11 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./jest-e2e.json",
```

Comment:
e2e tests are most likely need to be executed inside the docker container, so this command can be used by the container and another command needs to be created to trigger tests from outside the container:
```
  "docker:test:e2e": "test:e2e": "jest --config ./jest-e2e.json",
  "test:e2e": "docker exec -it 'npm' 'run' 'docker:test:e2e'",
```